src/dbnode/integration/peers_bootstrap_high_concurrency_test.go

```diff
@@ -23,33 +23,71 @@
 package integration
 
 import (
+	"encoding/json"
 	"fmt"
 	"testing"
 	"time"
 
 	"github.com/m3db/m3/src/dbnode/integration/generate"
 	"github.com/m3db/m3/src/dbnode/namespace"
 	"github.com/m3db/m3/src/dbnode/retention"
+	"github.com/m3db/m3/src/dbnode/storage/index"
+	"github.com/m3db/m3/src/m3ninx/idx"
+	"github.com/m3db/m3/src/x/ident"
 	xtest "github.com/m3db/m3/src/x/test"
 
 	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
 )
 
 func TestPeersBootstrapHighConcurrency(t *testing.T) {
+	for _, test := range []testPeersBootstrapHighConcurrencyOptions{
+		{BatchSize: 16, Concurrency: 64, BatchesPerWorker: 8},
```

Comment:
Nice, were there other configs worth trying?

Reply:
The tests take a substantial time, so perhaps I'll add just one more with a higher batch size and lower concurrency, maybe invert these numbers.

Reply:
Added one inverted test.